app/Models/Multiplayer/Game.php

```diff
@@ -19,24 +19,75 @@
  */
 namespace App\Models\Multiplayer;
 
+use App\Models\Beatmap;
+use App\Models\Mod;
+
 class Game extends Model
 {
     protected $table = 'games';
     protected $primaryKey = 'game_id';
+
     protected $hidden = ['match_id'];
+
     protected $dates = [
         'start_time',
         'end_time',
     ];
+
     public $timestamps = false;
 
+    const SCORING_TYPES = [
+        0 => 'score',
```

Comment:
as this way of lookup probably only happens when reading the attribute, better flip the array? so one can for example do `->where('scoring_type', static::SCORING_TYPES['accuracy'])`. The same for `TEAM_TYPES` below.